R/tm_t_crosstable.R

```diff
@@ -18,27 +18,72 @@
 #' @note For more examples, please see the vignette "Using cross table" via
 #'   `vignette("using-cross-table", package = "teal.modules.general")`.
 #'
-#' @export
-#'
 #' @examples
-#' # Percentage cross table of variables from ADSL dataset
+#' # general data example
+#' library(teal.widgets)
+#'
+#' data <- teal_data()
+#' data <- within(data, {
+#'   CO2 <- CO2
+#'   CO2[["primary_key"]] <- seq_len(nrow(CO2))
+#' })
```

Comment:
I suggest less correlated categorical variables. Previously used `mtcars` looks like better candidate. Then `x` could be all variables and `y` only factors 


```suggestion
#' data <- within(data, {
#'   library(nestcolor)
#'   mtcars <- mtcars
#'   for (v in c("cyl", "vs", "am", "gear")) {
#'     mtcars[[v]] <- as.factor(mtcars[[v]])
#'   }
#' })
```

Reply:
`mtcars` it is :+1: